batchflow/models/torch/base.py

```diff
@@ -559,6 +559,11 @@ def _get_placeholder_shapes(self):
                 if self.classes is None:
                     self.classes = shapes[0][0]
 
+    def _to_device(self):
+        if len(self.devices) > 1:
+            self.model = nn.DataParallel(self.model, self.devices)
```

Comment:
Do we always need data parallel with multiple devices?

Reply:
For now, I believe we do.
The only exception I see is when you want to run model parallel mode, e.g. you have have a model that doesn't fit into one device and you want to put it on several devices.